libnmstate/nm/sriov.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate.nm import connection as nm_connection
+from libnmstate.nm import nmclient
+from libnmstate.schema import Ethernet
+from libnmstate.schema import SRIOV
+
+
+def create_setting(iface_state, base_con_profile):
+    ethernet_setting = iface_state.get(Ethernet.CONFIG_SUBTREE)
+    if not ethernet_setting:
+        return None
```

Comment:
Is None correct here? Why not return `sriov_setting`? Then it could be simplified using `.get()` with `{}` as the second argument, see #592 .

Reply:
Done!